docs/server/auto-mode-service.md

```
@@ -0,0 +1,194 @@
+AutoModeService
+  ├── FeatureScheduler      — loop ownership, feature loading, dispatch
+  ├── ExecutionService      — per-feature execution logic (plan, build, verify, merge)
+  ├── ConcurrencyManager    — concurrency slot tracking across all projects
+  ├── AutoLoopCoordinator   — per-worktree loop state (running, paused, failures)
+  ├── FeatureStateManager   — persist-before-emit status transitions
+  └── TypedEventBus         — type-safe event emission
+```
+
+### Feature Execution Flow
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)